### PR TITLE
Fix install stage on traivs

### PR DESCRIPTION
**Pull request by sideeffffect**
this is the first step to fix travis builds
the build still fails though (#369)

### CODE REVIEW

**Comment by ferventcoder** [comment]
@sideeffffect Have you had an opportunity to read over our CONTRIBUTING document?

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: csharp
 install:
-  - sudo apt-get install mono-devel mono-gmcs
```
Comment (ferventcoder) [inline .travis.yml]: Does this mean that mono-devel is already there?

Comment (sideeffffect) [inline .travis.yml]: [indeed there is](https://travis-ci.org/chocolatey/choco/builds/72637212#L953)

**Comment by sideeffffect** [comment]
hi, I've just glossed over it now

 * this seems to be *trivial* change so we shouldn't need CLA; since this project is licensed under a permissive license, I would sign
 * I see I should have made a special name for the commit (`(maint)`), but I don't know how to rename it now

File: .travis.yml (hunk 1 above)
Comment (ferventcoder) [inline .travis.yml]: :+1:

**Comment by ferventcoder** [comment]
Read through the entire contributing document, it will explain how to fix your commit messages and repush the changes.

**Comment by ferventcoder** [comment]
I actually think we'll close this one for now in preference for #372 (and while we wait on the Xamarin bug to get evaluated). Thank you for researching that! 

I think the next step is to get the specs failure working, which is wide open for either of you to do if you are interested.

**Comment by ferventcoder** [comment]
I pushed the fixes for #372 with fixups to reference the bug you filed against Xamarin in the commit - see https://github.com/chocolatey/choco/commit/6c200e11e5b8b4be428ef6a85e80c5e138a7643a for details.

**Comment by ferventcoder** [comment]
I fixed a few of the specs. Let's see if that means we have a green Travis build.

**Comment by ferventcoder** [comment]
And we are green! 
image

**Pull request closed without merging**